anyway/parsers/location_extraction.py

```diff
@@ -106,15 +108,25 @@ def get_bounding_box(latitude, longitude, distance_in_km):
     query_obj = (
         db.session.query(AccidentMarkerView)
         .filter(AccidentMarkerView.geom.intersects(polygon_str))
-        .filter(AccidentMarkerView.accident_year >= 2014)
+        .filter(AccidentMarkerView.accident_year >= 2015)
```

Comment:
maybe have this determined dynamically? e.g. current year - some period? otherwise we will need to remember to update this every year

Reply:
I agree, @aentin16 can you modify it?
For example, it can be relative to current year or even better to latest year in DB since they don't always aligned - we get the data in delay of 2 months (see for example [this function](https://github.com/hasadna/anyway/blob/35dec531fd4ac79c99d09e684027df017e989ddc/anyway/models.py#L335))

Reply:
Good call. I did current year minus 6, is that ok?